charger/ocpp/connector.go

```diff
@@ -83,6 +83,27 @@ func (conn *Connector) TriggerMessageRequest(feature remotetrigger.MessageTrigge
 	})
 }
 
+func (conn *Connector) SetChargingProfile(profile *types.ChargingProfile) error {
+	return Instance().SetChargingProfileRequest(conn.cp.ID(), conn.ID(), profile)
+}
+
+// getScheduleLimit queries the current or power limit the charge point is currently set to offer
+func (conn *Connector) GetScheduleLimit(duration int) (float64, error) {
```

Comment:
Was ist denn hier anders als bei SetChargingProfileRequest? Warum landen die Funktion hier, die andere in cs_core?